src/components/smart_objects/src/smart_object.cc

```diff
@@ -298,6 +290,16 @@ bool SmartObject::operator==(const int64_t Value) const {
 }
 
 // =============================================================
+// uint64_t TYPE SUPPORT
```

Comment:
These comments are useless. the type is written in the arg type. No need to fix.